README.md

```diff
@@ -30,21 +30,235 @@
+- [Millions of transactions per second](https://postgresml.org/blog/scaling-postgresml-to-one-million-requests-per-second)
+- [Horizontal scalability](https://github.com/postgresml/pgcat)
+
+
+**Training a classification model**
+
+<table>
+<tr>
+<td> Training </td>
+<td> Inference </td>
+</tr>
+<tr>
+<td>
+
+
+```sql
+SELECT * FROM pgml.train(
+    'Handwritten Digit Image Classifier',
+    algorithm => 'xgboost',
+    'classification',
+    'pgml.digits',
+    'target'
+);
+```
+
+</td>
+<td>
+
+```sql
+SELECT pgml.predict(
+    'My Classification Project', 
+    ARRAY[0.1, 2.0, 5.0]
+) AS prediction;
+```
+</td>
+</tr>
+</table>
+
+## Installation
+PostgresML installation consists of three parts: PostgreSQL database, Postgres extension for machine learning and a dashboard app. The extension provides all the machine learning functionality and can be used independently using any SQL IDE. The dashboard app provides a eays to use interface for writing SQL notebooks, performing and tracking ML experiments and ML models.
+
+### Docker
 
-Please see the [quick start instructions](https://postgresml.org/user_guides/setup/quick_start_with_docker/) for general information on installing or deploying PostgresML. A [developer guide](https://postgresml.org/docs/guides/setup/developers) is also available for those who would like to contribute.
+Step 1: Clone this repository
+
+```bash
+git clone git@github.com:postgresml/postgresml.git
+```
+
+Step 2: Start dockerized services. PostgresML will run on port 5433, just in case you already have Postgres running. You can find Docker installation instructions [here](https://docs.docker.com/desktop/)
+```bash
+cd postgresml
+docker-compose up
+```
+
+Step 3: Connect to PostgresDB with PostgresML enabled using a SQL IDE or [`psql`](https://www.postgresql.org/docs/current/app-psql.html)
+```bash
+postgres://postgres@localhost:5433/pgml_development
+```
+
+### Free trial
+If you want to check out the functionality without the hassle of Docker please go ahead and start PostgresML by signing up for a free account [here](https://postgresml.org/signup). We will provide 5GiB disk space on a shared tenant.
+
+## Getting Started
+
+### Option 1
+- On local installation go to dashboard app at `http://localhost:8000/` and use SQL notebooks to get started.
+- On the free tier click on **Dashboard** button to use SQL notebooks
+![dashboard](pgml-docs/docs/images/dashboard.png)
+
+### Option 2
+- Use any of the following popular tools to connect PostgresML and write SQL queries.
+  - Apache Superset
+  - DBeaver
+  - Data Grip
+  - Tableau
+  - Power BI
+  - Jupyter
+  - VSCode
+
+## NLP Tasks
+- Text Classification
+- Token Classification
+- Table Question Answering
+- Question Answering
+- Zero-Shot Classification
+- Translation
+- Summarization
+- nConversational
+- Text Generation
+- Text2Text Generation
+- Fill-Mask
+- Sentence Similarity
+
+## Regression
+## Classification
+
+## Applications
+### Text
```

Comment:
Are you planning to expand all of these bullet lists? They seem fairly bare bones right now.

Reply:
That is correct. I will expand on these bullets.

Reply:
Expanded text classification.